CHANGELOG.md

```diff
@@ -10,10 +10,25 @@ Released: TBD
 
 ### Major Changes
 
+- None
+
 ### Minor Changes
 
+- [#261](https://github.com/peggyjs/peggy/pull/261): Remove documentation from
+  README.md, instead linking to the HTML documentation.  HTML documentation on
+  peggyjs.org is now served from the `docs` branch, so that it won't update as
+  we chaing the `main` branch.  `main` must be merged onto `docs` as a part of
+  the releaase process going forward.
```

Comment:
```suggestion
- [#261](https://github.com/peggyjs/peggy/pull/261): Remove documentation from
  README.md, instead linking to the HTML documentation.  HTML documentation on
  <https://peggyjs.org> is now served from the `docs` branch, so that it won't update as
  we change the `main` branch.  `main` must be merged onto `docs` as a part of
  the release process going forward.
```